platforms/ios/config.cmake

```diff
@@ -7,6 +7,7 @@ set(IOS TRUE)
 set(CMAKE_OSX_SYSROOT "iphoneos")
 set(CMAKE_XCODE_EFFECTIVE_PLATFORMS "-iphoneos;-iphonesimulator")
 set(CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET "9.3")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fvisibility=hidden -fvisibility-inlines-hidden")
```

Comment:
This will also reduce our library size (maybe not by much but still)

Reply:
We should also do the same on other platforms, specifically in android's `build.gradle`.

Reply:
Yeah I'm not sure exactly how this works for shared libraries, but worth looking into. It does trim a little bit off the build output size as well!